lib/usdUfe/ufe/UsdRootChildHierarchy.cpp

```diff
@@ -55,7 +55,7 @@ Ufe::SceneItem::Ptr UsdRootChildHierarchy::parent() const
     // Our parent path should be in a different runtime.
     assert(parentPath.runTimeId() != path().runTimeId());
 #endif
-    if (parentPath.runTimeId() != path().runTimeId())
+    if (parentPath.runTimeId() == path().runTimeId())
         TF_WARN(kNotGatewayNodePath, path().string().c_str());
```

Comment:
Can you change this to a more proper
```suggestion
        TF_WARN(kNotGatewayNodePath, Ufe::PathString::string(path()).c_str());
```

Reply:
Yes good idea. Done.